modules/microshift-updating-rpms-y.adoc

```diff
@@ -0,0 +1,67 @@
+//Module included in the following assemblies:
+//
+//*  microshift_updating/microshift-update-rpms.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-updating-rpms_{context}"]
+= Applying minor-version updates with RPMs
+
+Updating a {product-title} minor version on a non-OSTree system is a simple matter of downloading then updating the RPMs. For example, use the following procedure to upgrade from 4.14 to 4.15.
+
+[IMPORTANT]
+====
+You can only update {product-title} from one version to the next in sequence. Jumping minor versions is not supported. For example, must update 4.14 to 4.15.
+====
+
+.Prerequisites
+* The system requirements for installing {product-title} have been met.
+* You have root user access to the host.
+* The version of {product-title} you have is compatible to upgrade to the version you are preparing to use.
+* You have verified that your host operating system is compatible with the version of {product-title} you are preparing to install.
+* You have completed a system backup.
+//* You have stopped {product-title} and any associated workloads.
+
+[NOTE]
+====
+You cannot downgrade {product-title} with this process. Downgrades are not supported.
+====
+
+.Procedure
+
+. As a root user, enable the {product-title} repositories by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -m)-rpms \
```

Comment:
nit: page says: "use the following procedure to upgrade from 4.14 to 4.15" but `{ocp-version}` is "4.14", and we don't have 4.15

I guess there's no other way since we don't support update from 4.13?